Modules/posixmodule.c

```diff
@@ -5615,8 +5630,40 @@ os_posix_spawnp_impl(PyObject *module, path_t *path, PyObject *argv,
 }
 #endif /* HAVE_POSIX_SPAWNP */
 
+#ifdef HAVE_RTPSPAWN
+static intptr_t
+_rtp_spawn(int mode, const char *rtpFileName, const char *argv[])
+{
+     RTP_ID rtpid;
+     int status;
+
+     rtpid = rtpSpawn(rtpFileName, argv, (const char **)environ,
+                      100, 0x1000000, 0, VX_FP_TASK);
+     if ((rtpid != RTP_ID_ERROR) && (mode == _P_WAIT)) {
+         (void) waitpid((pid_t)rtpid, &status, 0);
+         return ((intptr_t)status);
```

Comment:
Does this match the [documented semantics of `os.spawnv`](https://docs.python.org/3/library/os.html#os.spawnv)?
> ...if mode is P_WAIT, returns the process’s exit code if it exits normally, or -signal, where signal is the signal that killed the process.

The raw status doesn't conform to the above on Linux.

Reply:
In P_WAIT mode, VxWorks will not return -signal in the abnormal case. It just returns -1.